src/index.html

```diff
@@ -2,15 +2,51 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="get"
+      class="search__bar search__bar--big"
+      data-qa="big">
```

Comment:
```suggestion
    <form
      action="#"
      method="get"
      class="search__bar search__bar--big"
      data-qa="big"
    >
```
Fix all cases
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96409563/5083a8bd-1247-4dc0-8d26-c11e4c5f6f0e)